### ISSUE TITLE
The Neural Network Potential in Parallel

### ISSUE THREAD

**Issue by LiMahappy** [openkim/kliff#179]
Hi mjwen@mjwen ,
I am currently attempting to develop a neural network potential for a PtRh alloy. I have trained using a 2000-step AIMD dataset with KLIFF, but the resulting neural network potential has very low accuracy and is essentially unusable. Below is my input file. Additionally, I noticed on the official website that only physically-based potential training can be run in parallel mode. How can I run the neural network potential in parallel?
[nn_ptrh.txt](https://github.com/openkim/kliff/files/15425951/nn_ptrh.txt)
Awaiting your reply.

**Comment by mjwen** [comment]
Hi @LiMahappy, 

The quality of a model depends on multiple factors, and you might want to generate a dataset that covers the physics of the problem you are interested in studying. 

What do you mean by running in parallel? Training the model in parallel or using the trained model in parallel for simulation? If the former, you can use pytorch [distributed data parallel](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html) (pytorch is what kliff used internally). If the latter, the simulator (e.g. lammps, gulp) will provide parallelization.

**Comment by LiMahappy** [comment]
Hi @mjwen ,
Sorry, I may not have made myself clear, I was talking about training a neural network potential. Physical potentials can be trained in parallel using Python's multiprocessing module or the mpi4py module, but can neural network potentials be modified from just the input file or the submit command in order to be trained in parallel?
